CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 3.8.2 (August 11, 2022)
+IMPROVEMENTS:
+* Add bootstrap field to Consul backend resources: ([#1571](https://github.com/hashicorp/terraform-provider-vault/pull/1571))
+* Add `data` field to KV data sources: ([#1577](https://github.com/hashicorp/terraform-provider-vault/pull/1577))
+* Update kvv1 test to use AssertVaultState(): ([#1574](https://github.com/hashicorp/terraform-provider-vault/pull/1574))
+
+BUGS:
+* fix: remove unnecessary nesting of secret data for KV-V1 secrets: ([#1570](https://github.com/hashicorp/terraform-provider-vault/pull/1570))
+
+BACKWARDS INCOMPATIBILITIES / NOTES:
```

Comment:
Should probably come at the beginning and I think we can just call it `NOTES` or `IMPORTANT NOTES`

Reply:
Updated, thanks!